stdlib/3/builtins.pyi

```diff
@@ -525,7 +525,7 @@ class slice:
     @overload
     def __init__(self, stop: Optional[int]) -> None: ...
     @overload
-    def __init__(self, start: Optional[int], stop: Optional[int], step: int = None) -> None: ...
+    def __init__(self, start: Optional[int], stop: int = None, step: int = None) -> None: ...
```

Comment:
Wasn't the previous code correct? The first overload takes care of the case where only one argument is given.